src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+class RightTriangle extends Figure {
```

Comment:
The class name 'RightTriangle' should not start with an empty line. Please remove the empty line before the class declaration.